src/modes/simple_select.js

```diff
@@ -10,6 +10,7 @@ module.exports = function(ctx, options = {}) {
   var startPos = null;
   var featureCoords = null;
   var startCoordinates = null;
+  var mapCanvas = ctx.map.getCanvasContainer();
```

Comment:
I'd prefer to not have refs to the map as part of the mode. This is the goal of the events.js. Adding a new handler to `mouseout` like we have for `keyup` and `keydown` would be better.